cmd/mqtt/main.go

```diff
@@ -36,14 +36,17 @@ const (
 	defLogLevel = "error"
 	envLogLevel = "MF_MQTT_ADAPTER_LOG_LEVEL"
 	// MQTT
-	defMQTTPort             = "1883"
-	defMQTTTargetHost       = "0.0.0.0"
-	defMQTTTargetPort       = "1883"
-	defMQTTForwarderTimeout = "30s" // 30 seconds
-	envMQTTPort             = "MF_MQTT_ADAPTER_MQTT_PORT"
-	envMQTTTargetHost       = "MF_MQTT_ADAPTER_MQTT_TARGET_HOST"
-	envMQTTTargetPort       = "MF_MQTT_ADAPTER_MQTT_TARGET_PORT"
-	envMQTTForwarderTimeout = "MF_MQTT_ADAPTER_FORWARDER_TIMEOUT"
+	defMQTTPort              = "1883"
+	defMQTTTargetHost        = "0.0.0.0"
+	defMQTTTargetPort        = "1883"
+	defMQTTForwarderTimeout  = "30s" // 30 seconds
+	healthCheckSleep         = 1 * time.Second
```

Comment:
It's OK not to have this configurable, but I'd extract it to a separate constant (`const mqttHealthCheckSleep = time.Second)`.